modules/securityhub/README.md

```diff
@@ -33,62 +33,55 @@ The example snippet below shows how to use this component:
 ```yaml
 components:
   terraform:
-    securityhub/common:
+    securityhub:
       metadata:
-        component: securityhub/common
+        component: securityhub
       vars:
         enabled: true
         account_map_tenant: core
         central_resource_collector_account: core-security
-        admin_delegated: false
         central_resource_collector_region: us-east-1
-        finding_aggregator_enabled: true
         create_sns_topic: true
         enable_default_standards: false
         enabled_standards:
           - standards/cis-aws-foundations-benchmark/v/1.4.0
+        admin_delegated: false
+        finding_aggregator_enabled: true
 ```
 
 ## Deployment
 
-1. Apply `securityhub/common` to all accounts
-2. Apply `securityhub/root` to `core-root` account
-3. Apply `securityhub/common` to `core-security` with `var.admin_delegated = true`
+In order to deploy AWS Security Hub to multiple accounts with single aggregation region.
+
+1. Deploy `securityhub` to every account except `core-security` and `core-root`. This will deploy Security Hub account resource.
+2. Deploy `securityhub` to `core-security` with `var.admin_delegated = false`. This will deploy Security Hub account resource into `core-security`
+3. Deploy `securityhub` to `core-root` (use `SuperAdmin` role). . This will deploy Security Hub into `core-root`.
+4. Deploy `securityhub` to `core-security` with `var.admin_delegated = true`. This will delegate `core-security` as a collector account in particular region so all findings will be reported into it.
```

Comment:
This is not reasonable. Order of operations should be:

1. Deploy to `core-root` using a role authorized to make changes in the root account. (Which roles are authorized will vary based from org to org based on how much the root account has been locked down.) This will deploy Security Hub account resource into `core-root` and delegate Security Hub administration to `core-security`
2. Deploy to `core-security`. This will designate `core-security` as the collector account for all regions so all findings will be reported into it. It will also enable Security Hub in all current accounts and set up new accounts to be automatically enabled.

If the above is not how it works, it should be. My understanding is that was the point of [awsutils_security_hub_organization_settings](https://registry.terraform.io/providers/cloudposse/awsutils/latest/docs/resources/security_hub_organization_settings#member_accounts).

Reply:
In 1.
> This will deploy Security Hub account resource into core-root and delegate Security Hub administration to core-security
This will deploy Security Hub into root account and as well will enable Security Hub in delegated (in our example `core-security`) account.

So it is better to deploy to delegated account (`core-security`) first and then deploy to root account.

I'm designating core-security as main collector account in step 4 when all other accounts are set up.